bin/oref0-pushover.sh

```diff
@@ -28,3 +28,76 @@ elif [[ $ONLYFOR =~ "insulin" ]] && ! cat $FILE | egrep "maxBolus"; then
 else
     curl -s -F "token=$TOKEN" -F "user=$USER" -F "sound=$SOUND" -F "message=$(jq -c "{bg, tick, carbsReq, insulinReq, reason}|del(.[] | nulls)" $FILE) - $(hostname)" https://api.pushover.net/1/messages.json && touch monitor/pushover-sent
 fi
+
+# Enhance to send to pushover glances API
+#   For watch face updates once every 10 minutes
+#   Works for apple watch complications
+#   Not tested fo android wear notifications yet
+
+# Use file to make sure we don't update glances more often than every 10 mins
+# Apple watch disables any complication updates that happen more frequently
+
+pushoverGlances=$(cat preferences.json | jq -M '.pushoverGlances')
+
+if [ "${pushoverGlances}" == "null" -o "${pushoverGlances}" == "false" ]; then
+    echo "No preference or false preference for pushoverGlances"
+  exit
+fi
+
+
+GLANCES="monitor/last_glance"
+GLUCOSE="monitor/glucose.json"
+if [ ! -f $GLANCES ]; then
+  # First time through it will get created older than 10 minutes so it'll fire
+  touch $GLANCES && touch -r $GLANCES -d '-11 mins' $GLANCES
```

Comment:
I think your first `touch` here is redundant.

Reply:
I'll remove that first touch. Also, I'm currently refactoring it to perform faster by reading values from smb-suggested.json only. Give me a day or two to make these changes and test it.